Makefile

```diff
@@ -1,7 +1,9 @@
-all:
-	go build .
 .PHONY: all
+all: fedora-coreos-stream-generator
+
+fedora-coreos-stream-generator: main.go go.mod Makefile
```

Comment:
In light of this change does it still make sense to add a `go get -u` line in the `make` command? I.e we will update fedora-coreos-stream-generator if go.mod changes, but we are updating it to the latest commit (as opposed to the latest tagged commit in go.mod).

Reply:
Isn't this missing all the other `.go` files though?

Reply:
> In light of this change, does it still make sense to add a `go get -u` line in the `make` command? I.e we will update fedora-coreos-stream-generator if go.mod changes, but we are updating it to the latest commit (as opposed to the latest tagged commit in go.mod).

It looks like `go get -u` matches the behaviour of Dependabot of sticking to tags if already on a tag. In that case, I think it'd make sense to have a `make update` target that runs `go get -u`. But I wouldn't make it part of the default build target.

Reply:
@jlebon Which other `.go` files?  There's only one in this repository.

Reply:
Bahh sorry, was looking at the wrong tab.

Reply:
> In that case, I think it'd make sense to have a make update target that runs go get -u.

good idea, will add this